### PR TITLE
Feature fix style#55 Katya Nemerovetska

### PR DESCRIPTION
**Pull request by Katy-dev**
dev
## Summary of issue

fixed style for User option component
added some test to registrationPage

## Testing approach

jest

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=90%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/components/forCabinetPage/userWorksSpace/userWorkSpace.module.scss [hunk 1]
```diff
@@ -55,6 +56,7 @@
 .mainMenu {
   width: 0rem;
   height: 50rem;
```
Comment (Rominos7) [inline src/components/forCabinetPage/userWorksSpace/userWorkSpace.module.scss]: Shoud we change height from 50rem to 55 rem as well??

**Review by andrew-melnychuk — DISMISSED** [review]
good

**Review by Romanarkko — APPROVED** [review]
Approve

**Review by Romanarkko — CHANGES_REQUESTED** [review]
Not approve, when i open image to change profile picture its shows wrong.

**Review by Romanarkko — APPROVED** [review]
Approve

**Review by andrew-melnychuk — APPROVED** [review]
good

**Review by Rominos7 — APPROVED** [review]
Good

**Pull request merged**